.github/workflows/cri_test.yml

```
@@ -68,3 +68,49 @@ jobs:
     - name: Cleaning
       if: ${{ always() }}
       run: ./scripts/github_runner/clean_cri_runner.sh
+
+  gvisor-cri-tests:
+    name: gVisor-CRI tests
+    env:
+      GITHUB_RUN_ID: ${{ github.run_id }}
+      GITHUB_VHIVE_ARGS: "-dbg"
+    runs-on: [self-hosted, cri]
+
+    steps:
+
+      - name: Host Info
+        run: |
+          echo $HOSTNAME
+          echo $GITHUB_RUN_ID
+
+      - name: Setup TMPDIR
+        run: mkdir -p $TMPDIR
+
+      - name: Set up Go 1.15
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
```

please rebase `feature/gvisor` branch to sync it up with the current main. we have updated Go version to 1.16 everywhere

```suggestion
      - name: Set up Go 1.16
        uses: actions/setup-go@v2
        with:
          go-version: 1.16
```